packages/desktop-client/src/components/NotesButton.tsx

```diff
@@ -1,116 +1,87 @@
   id: string;
   width?: number;
   height?: number;
   defaultColor?: string;
-  tooltipPosition?: TooltipPosition;
+  tooltipPosition?: ComponentProps<typeof Tooltip>['placement'];
   style?: CSSProperties;
 };
 export function NotesButton({
   id,
   width = 12,
   height = 12,
   defaultColor = theme.buttonNormalText,
-  tooltipPosition,
+  tooltipPosition = 'bottom start',
   style,
 }: NotesButtonProps) {
-  const [hover, setHover] = useState(false);
-  const tooltip = useTooltip();
-  const data = useLiveQuery(() => q('notes').filter({ id }).select('*'), [id]);
-  const note = data && data.length > 0 ? data[0].note : null;
+  const triggerRef = useRef(null);
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const data = useLiveQuery<NoteEntity[]>(
+    () => q('notes').filter({ id }).select('*'),
+    [id],
+  );
+  const note = data && data.length > 0 ? data[0].note : '';
   const hasNotes = note && note !== '';
 
-  function onClose(notes) {
-    send('notes-save', { id, note: notes });
-    tooltip.close();
-  }
-
-  const [delayHandler, setDelayHandler] = useState(null);
-
-  const handleMouseEnter = () => {
-    setDelayHandler(
-      setTimeout(() => {
-        setHover(true);
-      }, 300),
-    );
-  };
-
-  const handleMouseLeave = () => {
-    clearTimeout(delayHandler);
-    setHover(false);
-  };
+  const [tempNotes, setTempNotes] = useState<string>(note);
+  useEffect(() => setTempNotes(note), [note]);
 
-  // This account for both the tooltip hover, and editing tooltip
-  const tooltipOpen = tooltip.isOpen || (hasNotes && hover);
+  function onClose() {
+    send('notes-save', { id, note: tempNotes });
+    setIsOpen(false);
+  }
 
   return (
-    <View
-      style={{ flexShrink: 0 }}
-      onMouseEnter={handleMouseEnter}
-      onMouseLeave={handleMouseLeave}
+    <Tooltip
+      content={<Notes notes={note} />}
+      placement={tooltipPosition}
+      triggerProps={{
+        isDisabled: !hasNotes || isOpen,
+      }}
     >
-      <Button
-        type="bare"
-        aria-label="View notes"
-        className={!hasNotes && !tooltipOpen ? 'hover-visible' : ''}
-        style={{
-          color: defaultColor,
-          ...style,
-          ...(hasNotes && { display: 'flex !important' }),
-          ...(tooltipOpen && { color: theme.buttonNormalText }),
-        }}
-        {...tooltip.getOpenEvents()}
+      <View style={{ flexShrink: 0 }}>
+        <Button
+          ref={triggerRef}
+          type="bare"
+          aria-label="View notes"
+          className={!hasNotes && !isOpen ? 'hover-visible' : ''}
+          style={{
+            color: defaultColor,
+            ...style,
+            ...(hasNotes && { display: 'flex !important' }),
+            ...(isOpen && { color: theme.buttonNormalText }),
+          }}
+          onClick={() => setIsOpen(true)}
+        >
+          <SvgCustomNotesPaper style={{ width, height }} />
+        </Button>
+      </View>
+
+      <Popover
```

Comment:
So the `NotesButton` component now uses both Tooltip + Popover.

Tooltip - should be used for "hover" effects. They appear on hover. And disappear when moving mouse cursor away.

Popover - should be used for more permanent overlays that people can interact with (i.e. the textbox for editing notes).

Previously we were mixing these two concepts together, but I think it makes sense to slightly refactor this to separate the concerns in 2x different components.